### ISSUE TITLE
Question: Accessing schema at a specific Git commit

### ISSUE THREAD

**Issue by js-lowes** [kamilkisiela/graphql-hive#757]
I was wondering if it is currently possible to the latest sdl/date/metadata, however I could not find any way in the docs to get a list of all past sdl's. Is there anyway to get a list back like:
```json
[
  {
      "sdl": "...",
      "date": 1670362117635,
      "commit": "23i4oadnfkjn23hb5"
  },
  {
      "sdl": "...",
      "date": 1670362117634,
      "commit": "23i4oadnfkjn23hb5"
  },
]
```

Additionally be able to drill down and see of a specific commit exists and get the related data? If not via CDN does an API exist that devs could hit?

Thanks!!

**Comment by dotansimha** [comment]
At the moment the CDN only provides the latest valid schema, since that's the actual requirement of GraphQL gateways (SDL + metadata). 

I think storing every commit/schema and indexing it by identifier will require some architecture changes to the CDN structure and storage (not saying it's not possible, just stating the scope of changes needed to be done). @n1ru4l what do you think? 

Is it a requirement to have the complete history as part of the CDN? Today you can just access it as part of the GraphQL API (and fetched the Postgres database).

**Comment by js-lowes** [comment]
> Is it a requirement to have the complete history as part of the CDN? Today you can just access it as part of the GraphQL API (and fetched the Postgres database).

I think I missed this, did not know this was an option. Can you point where I can find that info on https://docs.graphql-hive.com/?

**Comment by kamilkisiela** [comment]
It will be solved by #2720
